conjure_oxide/src/rules/base.rs

```diff
@@ -1,22 +1,25 @@
 use conjure_core::{
     ast::Constant as Const, ast::Expression as Expr, metadata::Metadata, rule::RuleApplicationError,
 };
+use conjure_rule_sets::register_rule_set;
 use conjure_rules::register_rule;
 
 /*****************************************************************************/
 /*        This file contains basic rules for simplifying expressions         */
 /*****************************************************************************/
 
+register_rule_set!("Base", 50, ());
+
 /**
- * Remove nothings from expressions:
+ * Remove nothing's from expressions:
  * ```text
  * and([a, nothing, b]) = and([a, b])
  * sum([a, nothing, b]) = sum([a, b])
  * sum_leq([a, nothing, b], c) = sum_leq([a, b], c)
  * ...
  * ```
 */
-#[register_rule]
+#[register_rule(("Base", 50))] // ToDo: I am not sure what the priority should be - discuss
```

Comment:
Is this resolved?

Maybe some "magic numbers" might be helpful in the crate-level documentation?